deployer/src/config_creator.py

```diff
@@ -33,7 +33,14 @@ def extract_root_from_input(input_string):
         return domain.group() if domain else input_string
 
 
+def is_non_empty_list(is_non_empty_list):
```

Comment:
The function here raises an exception, it would be clearer for callers if its name was clear about it.
For example, let's call it: "assert_list_non_empty"